catalog.html

```diff
@@ -3,12 +3,18 @@
   <head>
     <meta charset="UTF-8" />
     <title>Каталог товаров</title>
+    <link rel="stylesheet" href="styles/styles.css" />
   </head>
 
   <body>
     <header class="page-header">
       <a href="index.html"
-        ><img src="" alt="Логитип интернет магазина Девайс" class="logo"
+        ><img
+          class="logo"
+          src="/img/icons/device-logo.svg"
```

Comment:
```suggestion
          src="img/icons/device-logo.svg"
```
Данный слеш не нужен (попробуйте посмотреть страницу в браузере - браузер изображение не находит). Подробнее [здесь](https://up.htmlacademy.ru/htmlcss/32/book/2.3) в параграфе "_Путь от папки в проекте_". Исправьте по всей разметке, где подключена графика.